components/PodcastContainer.jsx

```diff
@@ -9,32 +10,59 @@ class PodcastContainer extends React.Component {
       title: PropTypes.string,
       link: PropTypes.string,
     })).isRequired,
+    lastPlayedEpisode: PropTypes.string,
   };
 
+  static defaultProps = {
+    lastPlayedEpisode: undefined,
+  }
+
   constructor(props) {
     super(props);
     this.state = {
-      currentEpisodeLink: this.getLatestEpisode().link,
+      currentEpisodeLink: this.getEpisodeToPlay().link,
     };
   }
 
-  getLatestEpisode() {
-    const { episodes } = this.props;
+  getEpisodeToPlay() {
+    // eslint-disable-next-line react/prop-types
+    const { episodes, lastPlayedEpisode } = this.props;
+    if (lastPlayedEpisode) {
+      const titles = episodes.map(episode => episode.title);
+      const episodeIndex = titles.findIndex(
+        episodeName => episodeName === lastPlayedEpisode,
+      );
+      if (episodeIndex !== -1) {
+        return episodes[episodeIndex];
+      }
+    }
     return episodes[0];
   }
 
   updatePlayer = (newEpisode) => {
+    // eslint-disable-next-line no-undef
+    document.cookie = `LAST_PLAYED=${newEpisode.title}`;
```

Comment:
I think local storage might be a cleaner implementation, and wouldn't require any server changes.

`localStorage.setItem('LAST_PLAYED', newEpisode.title);`
`localStorage.getItem('LAST_PLAYED');`
`localStorage.removeItem('LAST_PLAYED');`

Reply:
Oh man, that was way easier! I had to move the call to componentDidMount, so that it would run on the client-side

Reply:
Nice!